spec/lib/swagger_v2_api_declaration.json

```diff
@@ -142,16 +152,6 @@
           "text/xml",
           "text/html"
         ],
-        "parameters": [
```

Comment:
Parameters are also still supported at the operation level, so removing these JSON data and removing the `parameter` blocks in the test below removes our current test coverage for that part of the spec. Please revert these changes, but keep the new things you added to test the new behavior. It's fine to let this JSON make less sense—wWe don't really care if this JSON is 100% correct or generates meaningful swagger data, just that it exercises all parts of the code we care about.